packages/grid/devspace.yaml

```diff
@@ -93,19 +101,24 @@ dev:
     labelSelector:
       app.kubernetes.io/name: syft
       app.kubernetes.io/component: backend
-    env:
-      - name: RELEASE
-        value: development
-      - name: DEV_MODE
-        value: "True"
-      - name: DEBUGGER_ENABLED
-        value: "True"
     ports:
       - port: "5678" # debugpy
-    sync:
-      - path: ./backend/grid:/root/app/grid
-      - path: ../syft:/root/app/syft
-    ssh: {}
+    containers:
+      backend-container:
+        env:
+          - name: RELEASE
+            value: development
+          - name: DEV_MODE
+            value: "True"
+          - name: DEBUGGER_ENABLED
+            value: "True"
+        sync:
+          - path: ./backend/grid:/root/app/grid
+          - path: ../syft:/root/app/syft
+      enclave-attestation:
+        sync:
+          - path: ./enclave/attestation/server:/app/server
+      ssh: {}
 
```

Comment:
MAYBE: could we add a commented out code in 
`values.dev.yaml`
like

![Screenshot 2024-04-16 at 4 55 47 PM](https://github.com/OpenMined/PySyft/assets/43314053/1db17937-4fdb-4e77-a98f-045a0ea84692)

So it is easy to re-enable it for testing by devspace

Reply:
Sure, that'll be helpful. Adding!